src/uTensor/core/types.hpp

```diff
@@ -12,6 +16,8 @@ class TensorShape {
   TensorShape(uint16_t shape1, uint16_t shape2, uint16_t shape3);
   TensorShape(uint16_t shape1, uint16_t shape2, uint16_t shape3,
               uint16_t shape4);
+
+  //FIXME:   array isn't avaliable on all embedded platforms
```

Comment:
std::array is super easy to write. I will do a custom one for uTensor

Reply:
std::* isn't popular among MCU the communities anyways. Don't terraform.

Reply:
Goal is to roll our own anyways, std:array actually has basically zero cost as nearly all the things it does are compile time and doesn't really generate any code. Basically just lowers to an array